### PR TITLE
Support markdownpreview

### PR DESCRIPTION
**Pull request by naokazuterada**
Hello @jonasbn 

This PR is associated with Issue #94.
Please check especially texts on README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -401,6 +402,45 @@ As mentioned you can disable this by setting the `uri_encoding` attribute to `fa
 # 中国的例子
 ```
 
+#### Markdown Preview compatible
+
+If you want to use this plugin with [Markdown Preview][MarkdownPreview], you should use `markdown_preview` attribute.
```
Comment (jonasbn) [inline README.md]: I would change "this plugin" to "**MarkdownTOC**"

File: README.md [hunk 2]
```diff
@@ -401,6 +402,45 @@ As mentioned you can disable this by setting the `uri_encoding` attribute to `fa
...
+You should set this attribute to `markdown` or `github`.
```
Comment (jonasbn) [inline README.md]: I would rephrase this to: "You can set this attribute to either ..."

File: README.md [hunk 3]
```diff
@@ -401,6 +402,45 @@ As mentioned you can disable this by setting the `uri_encoding` attribute to `fa
...
+- [ПРИМЕР RUSSIAN][%25D0%25BF%25D1%2580%25D0%25B8%25D0%25BC%25D0%25B5%25D1%2580-russian]
+
+<!-- /MarkdownTOC -->
+
+# Hello 世界 World
+# ESPAÑA
+# ПРИМЕР RUSSIAN
+```
+
+It doesn't support any other parsers.
```
Comment (jonasbn) [inline README.md]: I would rephrase this to: "Currently no other parsers are supported."

**Comment by naokazuterada** [comment]
@jonasbn Thank you for reviewing! I've modified in accordance with your comments.

**Comment by jonasbn** [comment]
Hi @naokazuterada 

Just let me know if there is anything else, my vacation last until the end of the week, then it is back to work :-) 

jonasbn

**Comment by naokazuterada** [comment]
Hi @jonasbn , sorry, I just want you to change your review status if all changes are OK and there are nothing else to change.
But it's OK now. I've already merged. Thank you for your support!
